migrations.lock

```diff
@@ -594,7 +594,6 @@ hqwebapp
  0009_truncate_authtoken_table
  0010_maintenancealert_scheduling
  0011_add_new_columns_and_rename_model
- 0012_rename_commcarehqalert_alert
```

Comment:
Missed updating this in commit earlier, https://github.com/dimagi/commcare-hq/pull/33592/commits/027ead6bfd418949894569bd73b18cd6867b8106